tests/gui/builder/control/test_selector.py

```diff
@@ -69,3 +71,19 @@ def test_selector_builder_3(gui: Gui, test_client, helpers):
         "value={_TpLv_tpec_TpExPr_selected_obj_TPMDL_0}",
     ]
     helpers.test_control_builder(gui, page, expected_list)
+
+
+def test_selector_builder_4(gui: Gui, test_client, helpers):
```

Comment:
I would have renamed this test to `test_selector_pandas_series`, which really is the point of it.

Reply:
The function name sounds much better. I updated it.